source/mainnet/net/resources/glossary.rst

```diff
@@ -309,6 +330,29 @@ in concert with the identity provider.
 
 Initial accounts are only relevant for Desktop Wallet and |mw-gen1|.
 
+.. _glossary-initialize:
+
+Initialize
+==========
+
+Action that creates a new smart contract instance with the initial state. The only way to update the instance state is by invoking the instance’s entrypoints.
+
+.. _glossary-instance:
+
+Instance
+========
+
+A smart contract module together with a specific state and an amount of CCD tokens. Multiple smart contract instances can be created from the same module. Smart contract instances can be created from a deployed :ref:`smart contract
+module<contract-module>` via the ``init`` transaction which invokes the requested function in the smart contract module. This function can take a parameter.Its end result is required to be the initial smart contract state of the
+instance.
+
+.. _glossary-invoke:
+
+Invoke
+======
+
+The act of triggering a receive function in a smart contract and viewing its return value. Invoking an instance is not a transaction and it does not change the state of a contract. Invoking can be useful to either view information about the instance or to test a receive method before running an update. Because invoking is not a transaction, there is no fee to run invoke.
```

Comment:
I think you could explicitly mention the idea of having view functions like here http://localhost:8000/smart-contracts/tutorials/nft-minting/mint-xfer.html?highlight=view#view-function
I think this gives some details on the " view information about the instance" part.